### ISSUE TITLE
Does it work with different CUDA and pytorch versions?

### ISSUE THREAD

**Issue by kamadforge** [ZiqiaoPeng/SyncTalk#181]
Did it work for you with any other version than
pip install torch==1.12.1+cu113 torchvision==0.13.1+cu113 torchaudio==0.12.1 --extra-index-url https://download.pytorch.org/whl/cu113
?

I have cuda 12.4 on Ubuntu 22.04 and when I installed

conda install pytorch torchvision torchaudio pytorch-cuda=12.4 -c pytorch-nightly -c nvidia

I have issues installing freqencoder, shencoder, etc.
I'm not sure if the encoders cannot compile because of this different cuda versions but perhaps anyone has experienced compiling them with a different cuda version

**Comment by hungho77** [comment]
install cuda 11.3 for pytorch 3d, after that you install newest cuda and pytorch version for remaining install

**Comment by kamadforge** [comment]
Interestingly, pytorch 3d ran fine with this set-up but the packages produce an error:
ERROR: Failed building wheel for freqencoder
etc.

pip install ./freqencoder
pip install ./shencoder
pip install ./gridencoder
pip install ./raymarching

Are thete any particular CUDA/pytorch requirments for these packages?

**Comment by hungho77** [comment]
yes, after install the pytorch3d, you have to reinstall pytorch match with your cuda toolkit version, then install above package
